hapi/hedera-protobufs/services/consensus_topic_info.proto

```diff
@@ -87,4 +88,37 @@ message ConsensusTopicInfo {
      * The ledger ID the response was returned from; please see <a href="https://github.com/hashgraph/hedera-improvement-proposal/blob/master/HIP/hip-198.md">HIP-198</a> for the network-specific IDs. 
      */
     bytes ledger_id = 9;
+
+    /**
+     * Access control for update/delete of custom fees.
+     * <p>
+     * If unset, custom fees CANNOT be set for this topic.<br/>
+     * If not set when the topic is created, this field CANNOT be set via update.
+     * If set when the topic is created, this field MAY be changed via update.
+     */
+    Key fee_schedule_key = 10;
+
+    /**
+     * A set of keys that are allowed to submit messages to the topic without
+     * paying the topic's custom fees.
+     * <p>
+     * If a submit transaction is signed by _any_ key included in this set,
+     *  custom fees SHALL NOT be charged for that transaction.
+     * <p>
+     * If fee_exempt_key_list is unset, the following keys are exempt
+     *  from custom fees: adminKey, submitKey, fee_schedule_key.
+     */
```

Comment:
Based on recent discussion, this clause should be removed (the list will not be filled in or interpreted in update).
```suggestion
     */
```

Reply:
Done